src/assets/src/components/preferences.tsx

```diff
@@ -69,25 +78,90 @@ function PreferencesEditor(props: PreferencesEditorProps) {
             label="As a host, I want to be notified via SMS when someone joins my empty queue." />
     );
 
-    const validateAndSubmit = (e: React.SyntheticEvent) => {
+    const oneTimePasswordTimer = () => {
+        let timer = process.env.REACT_APP_OTP_TIMER ? parseInt(process.env.REACT_APP_OTP_TIMER) : 30;
```

Comment:
Hi, @andcarls : I found the latest code not functional after this configuration change. There is no OTP code sent out, and code just stop after this line. I have added the REACT_APP_OTP_TIMER setting into .env file.

Could you please double check? Is there any local change that has not been checked in yet?

Reply:
Hmm, it appears that the error thrown is that "process" doesn't exist. I did some digging, and found that it is a bit more of a complex process to pass environment variables to React using webpack, requiring modifying the webpack scripts (in package.json) and the likely use of external modules (see [Dotenv](https://www.npmjs.com/package/dotenv)). I tried implementing some of the solutions I found online, but keep running into errors. Especially, for some odd reason, that webpack can't find the Dotenv module even after installation.

A simple solution would be to modify `webpack.config.js` to include:
```
plugins: [
        ...,
        new webpack.DefinePlugin({
            'process.env': {
                REACT_APP_OTP_TIMER: JSON.stringify(30),
            }
        })
    ]
```
Only changes a few lines and confirmed to work, but a bit less useful than a `.env` file.

I have never had to forward env variables to the frontend through webpack, and am running into weird errors locally. Any suggestions or does anyone have any insights on how to do this?